zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -16,6 +17,7 @@
 REPLICASET_TYPE = 'kube_replicaset'
 STATEFULSET_TYPE = 'kube_statefulset'
 DAEMONSET_TYPE = 'kube_daemonset'
+POSTGRESDB_TYPE = 'postgresql_database'
```

Comment:
`database` is already a very distinct type (well, two, for vintage stuff and RDS), I wouldn't mix in just another one.